nuxeo-features/nuxeo-annotation/nuxeo-annotation-core/src/main/resources/OSGI-INF/annotation-core-types-contrib.xml

```diff
@@ -10,6 +10,7 @@
   <extension target="org.nuxeo.ecm.core.schema.TypeService" point="doctype">
 
     <doctype name="Annotation" extends="Document">
+      <schema name="dublincore"/>
```

Comment:
Are you using dublincore ?  I missed it in the tests.

Reply:
We do, the annotations are now created with system user. I'll add a test.

Reply:
I can't easily test this aspect because of the whole login stack in unit tests. As it's a common behavior and already tested, I'll leave this PR as it.
Unless someone has an idea ?